PROJECT

```diff
@@ -20,4 +20,12 @@ resources:
     defaulting: true
     validation: true
     webhookVersion: v1
+- api:
```

Comment:
maybe we are missing some config here? 

https://github.com/grafana/loki/blob/main/operator/PROJECT#L1

Reply:
https://book.kubebuilder.io/reference/project-config.html